kibbeh/src/ui/SettingsDropdown.tsx

```diff
@@ -81,6 +82,20 @@ export const SettingsDropdown: React.FC<{
             transition
             onClick={() => setDebugAudio(!debugAudio)}
           />
+
+            <a
+            href="/download"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <SettingsIcon
+              onClick={onCloseDropdown}
```

Comment:
why does this not actually take you to the download page?

Reply:
I ran the web app locally, and everything works fine

Reply:
does the download button take you to the download page? *(i.e https://dogehouse.tv/download)*

Reply:
yup